edgelet/hsm-sys/azure-iot-hsm-c/CMakeLists.txt

```diff
@@ -186,3 +186,8 @@ set (CPACK_RPM_PACKAGE_LICENSE "Proprietary")
 
 include (CPack)
 
+if (${run_unittests})
```

Comment:
Why do you need this?

Reply:
Philip had this change in his iotedge fork so I inherited that to my fork. @philipktlin ?

Reply:
If I remember correctly, this is to avoid Checkin/CI build fail.  Try remove it and see if Checkin/CI build pass.

Reply:
checking...

Reply:
revert it for now and will kick off a build to verify

Reply:
Without this new section, below error is hit multiple times, SHFileOperation is from Shell32.lib, so looks like we need to keep the change.

```
test_utils.obj : error LNK2019: unresolved external symbol **__imp_SHFileOperationA** referenced in function hsm_test_util_delete_dir [I:\iotedge-1\edgelet\target\thumbv7a-pc-windows-msvc\release\build\hsm-sys-39e8967a78d2a48b\out\build\tests\edge_hsm_util_int\edge_hsm_util_int_exe.vcxproj]
I:\iotedge-1\edgelet\target\thumbv7a-pc-windows-msvc\release\build\hsm-sys-39e8967a78d2a48b\out\build\tests\edge_hsm_util_int\Release\edge_hsm_util_int_exe.exe : fatal error LNK1120: 1 unresolved externals [I:\iotedge-1\edgelet\target\thumbv7a-pc-windows-msvc\release\build\hsm-sys-39e8967a78d2a48b\out\build\tests\edge_hsm_util_int\edge_hsm_util_int_exe.vcxproj]
```

Reply:
Oh right, I remember finding this on Philip's machine. We concluded that the amd64 SDK was pulling in shell32.lib implicitly but arm SDK was not.